### DISCUSSION TITLE
Android车机平台使用PAGImageView/PAGView时如何在页面切抑或者隐藏时释放内存

### DISCUSSION THREAD

**Discussion by jterrence** [Tencent/libpag#2380] [General] [answered]
Android车机平台使用PAGImageView/PAGView时如何在页面切抑或者隐藏时释放内存，当前切换或隐藏时调用是pause方法，但还是不能及时释放内存资源

**Comment by jterrence** [comment]
另外咨询一下使用PAG动画库，是否会影响车机ION内存，如果有影响，如果及时释放或者是优化

**Comment by domchen** [comment, marked as answer]
freeCache方法就是干这个的。
